client/src/actions/index.js

```diff
@@ -318,6 +318,9 @@ export function getAlerts() {
     return dispatch => {
         promise.then(
             res => {
+                res.data.map(function(alert) {
+                    alert.created_at = new Date(alert.created_at).toLocaleString()
```

Comment:
I think we should use [moment](http://momentjs.com/) for the conversion because I feel like we could run into consistency problems using the built in date `Date`.